live-server.js

```diff
@@ -59,6 +60,13 @@ for (var i = process.argv.length - 1; i >= 2; --i) {
 		opts.logLevel = 0;
 		process.argv.splice(i, 1);
 	}
+	else if (arg.indexOf("--mount=") > -1) {
+		// e.g. "--mount=/components:./node_modules" will be ['/components', '<process.cwd()>/node_modules']
+		var mountRule = arg.substring(8).split(":");
+		mountRule[1] = path.resolve(process.cwd(), mountRule[1]); 
```

Comment:
Isn't it wrong to resolve to cwd here? Cwd isn't set to the final location (if given as a param) until after this param handling loop (at around line 90), and you do resolve a second time when actually creating the server.